src/locales/scripts/jed1x-json-to-json.js

```diff
@@ -0,0 +1,69 @@
+const set = require('lodash.set')
+
+/**
+ * Convert a Jed1x-Translate object to a nested JSON object.
+ *
+ * Create a nested JSON object, clean up keys from jed1x context with special
+ * character, remove keys without values, convert values from array to string,
+ * if strings are for plural forms, join them with the pipe character.
+ * Go from this:
+ * {
+ *  "browse-page.load\u0004Load more results": [
+ *     "Загрузить ещё результаты"
+ *   ],
+ *   "browse-page.all-result-count-more\u0004Over ###localeCount### results": [
+ *     "Более ###localeCount### результата",
+ *     "Более ###localeCount### результатов",
+ *     "Более ###localeCount### результатов"
+ *   ]
+ *  "browse-page.search-form.button\u0004Search": [],
+ * }
+ * To:
+ * {
+ *  "browse-page: {
+ *   "load": "Загрузить ещё результаты"
+ *   "all-result-count-more": "Более ###localeCount### результата|Более ###localeCount### результатов|Более ###localeCount### результатов",
+ * }
+ *
+ */
+
+// special character, context delimiter in jed format:
+// https://github.com/messageformat/Jed/blob/351c47d5c57c5c81e418414c53ca84075c518edb/jed.js#L117
+const SPLIT_CHAR = String.fromCharCode(4)
+
+function jed1xJsonToJson(jed1xObject) {
+  const result = {}
+  Object.entries(jed1xObject?.locale_data?.messages).forEach(([key, value]) => {
+    const cleanedKey = key.slice(0, key.indexOf(SPLIT_CHAR))
+    if (value.length > 0) {
+      const cleanedValue = value.length === 1 ? value[0] : value.join('|')
+      return set(result, cleanedKey, cleanedValue)
```

Comment:
Is `lodash.set` necessary here? Could we use the native `Object.assign` instead?

```suggestion
      return Object.assign(result, { [cleanedKey]:  cleanedValue })
```

Reply:
To be honest, I just adapted the code  that Zack wrote for NGX format. I think it converts the nested keys that have several dots in them into a nested object, but I'm not sure 🤔

Reply:
Ohhhh that makes sense, the `cleanedKey` is a path 😭 Never mind then!